### ISSUE TITLE
M1, Ex1, T1, step 7: To create queries, check the following tables. Please rephrase.

### ISSUE THREAD

**Issue by Theelepel** [MicrosoftLearning/DA-100-Analyzing-Data-with-Power-BI#135]
# Module: 01
#### Step: 07

Description of issue
"To create queries, check the following tables:"

Please rephrase this to something like "Tick the boxes". Currently it can confuse non native speakers; check = "just looking at it"

**Comment by shannonlindsay** [comment]
@Theelepel - I have updated the statement to say: " To create queries, select the checkbox next to the following six tables:" - does that improve the clarity?

**Comment by Theelepel** [comment]
Yes, great, that would avoid misunderstandings.

Thanks 😊
